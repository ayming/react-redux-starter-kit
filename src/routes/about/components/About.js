import React, { PureComponent } from 'react'

import Link from '../../../components/Link'
import s from './About.scss'

class About extends PureComponent {
  render() {
    return (
      <div className={s.wrapper}>
        <p>
          This starter kit is designed to get you up and running with a bunch of
          awesome front-end technologies.
        </p>
        <Link to="/">HOME</Link>
      </div>
    )
  }
}

export default About
