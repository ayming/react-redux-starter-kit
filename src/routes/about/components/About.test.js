/* eslint-env jest */
import React from 'react'
import renderer from 'react-test-renderer'

import About from './About'

describe('About', () => {
  test('renders correctly', () => {
    const wrapper = renderer.create(<About />).toJSON()

    expect(wrapper).toMatchSnapshot()
  })
})
