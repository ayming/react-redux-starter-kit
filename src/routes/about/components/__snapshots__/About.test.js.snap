// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders correctly 1`] = `
<div
  className="wrapper"
>
  <p>
    This starter kit is designed to get you up and running with a bunch of awesome front-end technologies.
  </p>
  <a
    href="/"
    method="push"
    onClick={[Function]}
  >
    HOME
  </a>
</div>
`;
